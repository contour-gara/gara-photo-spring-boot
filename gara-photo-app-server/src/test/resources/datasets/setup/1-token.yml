token:
  - id: 1
    access_token: accessToken
    refresh_token: refreshToken
    date_time: "2024-04-14 19:48:34"
