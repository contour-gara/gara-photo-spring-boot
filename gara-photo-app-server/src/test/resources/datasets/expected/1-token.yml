token:
  - id: 1
    access_token: accessToken
    refresh_token: refreshToken
    date_time: "2024-04-14T19:48:34+09:00[Asia/Tokyo]"
